moped-editor/src/views/projects/projectView/ProjectContractsTable.js

```diff
@@ -90,6 +92,17 @@ const ProjectContractsTable = () => {
       title: "Description",
       field: "description",
     },
+    {
+      title: "Work Assignment ID",
```

Comment:
not sure if this should be Work assignment ID to match the sentence casing, and also how it was on the project summary page. 

When I checked the project summary page, I noticed "eCAPRIS Subproject ID" isnt sentence case because of the S

Reply:
That is a good question, I'm thinking sentence case and maybe patch the eCAPRIS one? Not a super strong opinion from me but that is a good detail to catch for consistency!